src/gui/accountstate.cpp

```diff
@@ -223,14 +222,14 @@ void AccountState::setNavigationAppsEtagResponseHeader(const QByteArray &value)
     _navigationAppsEtagResponseHeader = value;
 }
 
-QString AccountState::notificationStatus() const
+bool AccountState::isDesktopNotificationsAllowed() const
 {
-    return _notificationStatus;
+    return _isDesktopNotificationsAllowed;
 }
 
-void AccountState::setNotificationStatus(const QString &status)
+void AccountState::setDesktopNotificationsStatus(const bool status)
```

Comment:
@camilasan This should be named `setDesktopNotificationsAllowed` similarly to how the getter is named `isDesktopNotificationsAllowed`

Reply:
Done.